torch/hub.py

```diff
@@ -165,6 +165,7 @@ def _get_cache_or_reload(github, force_reload):
         _remove_if_exists(extracted_repo)
         # Unzip the code and rename the base folder
         cached_zipfile.extractall(hub_dir)
+        cached_zipfile.close()
```

Comment:
Don't do this. ZipFile can be used as a context manager; that will ensure it is closed even if an exception is thrown.